supervision/detection/line_counter.py

```diff
@@ -141,8 +145,8 @@ def annotate(self, frame: np.ndarray, line_counter: LineZone) -> np.ndarray:
             lineType=cv2.LINE_AA,
         )
 
-        in_text = f"in: {line_counter.in_count}"
-        out_text = f"out: {line_counter.out_count}"
+        in_text = f"in: {line_counter.in_count}" if len(self.custom_in_text) == 0 else f"{self.custom_in_text}: {line_counter.in_count}"
```

Comment:
Then here we would have `if self.custom_in_text is not None`. Same for other text.

Reply:
I like the idea. I left two comments. ;)

Reply:
Thanks, I applied your comments, lmk if you see something that can be improved too. 👋🏽